### ISSUE TITLE
Unable to find workflow 'Graphs CI'

### ISSUE THREAD

**Issue by o3o-ca** [upptime/upptime#995]
**Description:**  

I followed the [Getting Started](https://upptime.js.org/docs/get-started) guide and created a [repository](https://github.com/o3o-ca/o3o-uptime) using the Upptime template, ensuring that all branches were included. I added the `GH_PAT` secret and then updated and pushed the `.upptimerc.yml` file to configure the sites and CNAME for the status page. The default branch is `master`.

Editing the `.upptimerc.yml` file was the only changes made, which are reflected in this commit: [9e15d76](https://github.com/o3o-ca/o3o-uptime/commit/9e15d76a7519f38d6d850afc076a29026fefa168).

However, it seems that the GitHub Action was triggered immediately upon repository creation, before I could add the `GH_PAT` secret, resulting in a failure. This probably should not have happened, as no workflows should have been run before any configuration changes were made.

When I manually re-triggered the "Setup CI" workflow, I encountered the following error:

```
Error: Unable to find workflow 'Graphs CI' in o3o-ca/o3o-uptime 😥
```

Full log: [job-logs.txt](https://github.com/user-attachments/files/16734809/job-logs.txt)

![image](https://github.com/user-attachments/assets/0b2357c3-9c98-4b75-941e-015505dbd72d)

I also noticed that in the Actions tab, only `pages-build-deployment` and `Setup CI` workflows are listed. Despite following the guideline and recreating the repository, the error persists whenever I run "Setup CI". 

![image](https://github.com/user-attachments/assets/cd4196f3-67ad-4830-9301-4445cc21ebe1)

**Steps to Reproduce:**
1. Follow "Getting Started" guide for Upptime.
2. Create repository from template, including all branches.
3. Add `GH_PAT` secret.
4. Update `.upptimerc.yml` with sites and CNAME.
5. Attempt to re-run the already failed "Setup CI" workflow.

**Expected Behavior:**
No workflows should be triggered automatically before adding the `GH_PAT` secret and configuring `.upptimerc.yml`. The "Setup CI" workflow should run without errors.

**Actual Behavior:**
The workflow was triggered automatically before making any changes, and the "Setup CI" workflow fails with an error: "Unable to find workflow 'Graphs CI'."

**Environment:**
- Repository: [o3o-uptime](https://github.com/o3o-ca/o3o-uptime)

Please advise on how to resolve this issue. Thank you!

**Comment by windstatus** [comment]
Same issue for me. The actions ran **immediately** after the repo was created and failed.

**Comment by flotwig** [comment]
Same here: https://github.com/flotwig/status.bloomqu.ist/actions/runs/10549724017

I just set up Upptime. Seems to be causing my status website to not update with my customizations: https://status.bloomqu.ist/

**Comment by NilsBaumgartner1994** [comment]
Fixed it by correctly setting the token settings.

**Comment by kaerez** [comment]
Same issue here as well :-(
https://github.com/kaerez/upptime

**Comment by flotwig** [comment]
The issue seems to have resolved itself without me doing anything :smile:

**Comment by Descent098** [comment]
I had to modify the file in `.github/workflows/graphs.yml` (in my case I added a [single space](https://github.com/Descent098/up/commit/45b20e977dcdebbd619485c5d8c9d74c6d7c5146)). Once I modified it github recognized the file as a workflow file. So I'm guessing this is a github actions bug with however it's finding files

**Comment by finalgamer** [comment]
Thanks @Descent098. I did the same thing. [Add a space](https://github.com/Nau-media-AG/uptime/commit/5e7f5323c86db712b9849d0633055bd2f0afcfb3) and it worked

**Comment by SphenHe** [comment]
> Fixed it by correctly setting the token settings.

I checked the log of `Setup Upptime` job and looked into the `Update Summary in README` part though GitHub actions said it was  "✅", and I found this error:

```log
remote: Permission to SphenHe/upptime.git denied to SphenHe.
fatal: unable to access 'https://github.com/SphenHe/upptime/': The requested URL returned error: 403
Found recently closed issues 0
```

And maybe that's why I cannot `find workflow 'Graphs CI' in SphenHe/upptime`. When I generate a new token with [right permission](https://upptime.js.org/docs/get-started/#add-repository-secrets), this issue is solved.

Notice that if you delete your upptime repo and create a newer one with the same name, you cannot reuse the secret and must generate a new one.

**Comment by fboerman** [comment]
for me too the space adding fixed the issue!

**Comment by Runkang10** [comment]
Wow! Just add a space in graphs.yml to fix the issue.

**Comment by omarcostahamido** [comment]
sounds like a pretty solid solution from @Descent098 
thank you very much!
I've opened PR to try to save other newcomers some headache as well.

**Comment by finalgamer** [comment]
@omarcostahamido I don't think that #1001 fixes the core issue here. I assume that because the file was edited GitHub reindexed the action. The core issue might be a problem with GitHub.

**Comment by yanyao2333** [comment]
Thanks @Descent098 . I added a space, and it was fixed! (But I still don't know why)

**Comment by Descent098** [comment]
@yanyao2333  no worries, I think @finalgamer is correct. I'm guessing the initial method github is using to find the workflow files is failing on initialization, but adding the space basically is forcing a cache-reset, which makes it visible to the actions system
